test/studies/labelprop/labelprop-tweets.chpl

```diff
@@ -187,10 +186,34 @@ proc process_json(logfile:channel, fname:string, ref Pairs) {
 
   if progress then
     writeln(fname, " : processing");
- 
+
   while true {
-    got = logfile.readf("%~jt", tweet, error=err);
-    if got && !err {
+    try! {
+      got = logfile.readf("%~jt", tweet);
```

Comment:
if `readf` throws, then `got` won't be initialized here. I think you need to set it to `false` before the call here or as the first thing in the catch block.